pkg/alertmanager/api.go

```diff
@@ -130,8 +131,12 @@ func (am *MultitenantAlertmanager) DeleteUserConfig(w http.ResponseWriter, r *ht
 	w.WriteHeader(http.StatusOK)
 }
 
-func validateUserConfig(cfg alerts.AlertConfigDesc) error {
-	// Validation copied from: https://github.com/prometheus/alertmanager/blob/8e861c646bf67599a1704fc843c6a94d519ce312/cli/check_config.go#L65-L96
+// Partially copied from: https://github.com/prometheus/alertmanager/blob/8e861c646bf67599a1704fc843c6a94d519ce312/cli/check_config.go#L65-L96
+func validateUserConfig(logger log.Logger, cfg alerts.AlertConfigDesc) error {
+	if cfg.RawConfig == "" {
```

Comment:
Can we add a comment that we use the fallback config differently with this API, where we will provision an AM if you don't have any config. Hence we don't need to create an empty config here.

Reply:
done!